source/extensions/filters/http/grpc_json_transcoder/BUILD

```diff
@@ -14,10 +14,20 @@ envoy_extension_package()
 
 envoy_cc_library(
     name = "json_transcoder_filter_lib",
-    srcs = ["json_transcoder_filter.cc"],
-    hdrs = ["json_transcoder_filter.h"],
+    srcs = [
+        "async_reflection_fetcher.cc",
+        "descriptor_pool_builder.cc",
+        "json_transcoder_filter.cc",
+    ],
+    hdrs = [
+        "async_reflection_fetcher.h",
+        "async_reflection_fetcher_callbacks.h",
+        "descriptor_pool_builder.h",
```

Comment:
making two separate libraries  for descriptor_pool_builder and asyn_reflection_fetcher

Reply:
Will do.